src/qtiRunner/core/Renderer.js

```diff
@@ -647,25 +645,27 @@ const Renderer = function(options) {
                     this.shuffledChoices[serial] = [];
                     for (i = 0; i < 2; i++) {
                         choices[i] = interactionHelper.shuffleChoices(interaction.getChoices(i));
-                        this.shuffledChoices[serial][i] = _.pluck(choices[i], 'serial');
+                        this.shuffledChoices[serial][i] = choices[i].map(val => val.serial);
                     }
                 } else {
                     choices = interactionHelper.shuffleChoices(interaction.getChoices());
-                    this.shuffledChoices[serial] = _.pluck(choices, 'serial');
+                    this.shuffledChoices[serial] = choices.map(val => {
+                        val.serial;
+                    });
```

Comment:
issue: either return statement or braces removal is needed for it to work correctly
```suggestion
                    this.shuffledChoices[serial] = choices.map(val => val.serial);
```